study-builder/studies/mpc/emails/join_mailing_list.html

```diff
@@ -0,0 +1,286 @@
+                Thank you for signing up for the Metastatic Prostate Cancer Project mailing list.
+            </div>
+            <table border="0" cellpadding="0" cellspacing="0" class="wrapper" width="500">
+                <!-- LOGO/PREHEADER TEXT -->
+                <tr>
+                    <td align="center" class="logo" style="padding: 20px 0px 30px 0px;">
+                        <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                            <tr>
+                                <td align="center" bgcolor="#ffffff" width="100">
+                                    <a href="{{baseWebUrl}}" target="blank">
+                                        <img alt="Logo"
+                                             border="0" height="69"
+                                             src="https://storage.googleapis.com/${assetsBucketName}/mpc/project-logo.png"
+                                             style="display: block; font-family: Helvetica, Arial, sans-serif; color: #666666; font-size: 16px;"
+                                             width="150"/></a>
+                                </td>
+                            </tr>
+                        </table>
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+</table>
+<!-- ONE COLUMN SECTION -->
+<table border="0" cellpadding="0" cellspacing="0" width="100%">
+    <tr>
+        <td align="center" bgcolor="#ffffff" class="section-padding" style="padding: 15px 15px 15px 15px;">
+            <table border="0" cellpadding="0" cellspacing="0" class="responsive-table" width="500">
+                <tr>
+                    <td>
+                        <!-- COPY -->
+                        <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                            <tr>
+                                <td align="left" class="padding-copy"
+                                    style="padding: 30px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;">
+                                    Thank you for signing up for the Metastatic Prostate Cancer Project mailing list. By
+                                    being on this list, you will receive regular updates on our work and the
+                                    transformative research we hope to enable.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td align="left" class="padding-copy"
+                                    style="padding: 20px 0 0 0; font-size: 16px; line-height: 25px; font-family: Helvetica, Arial, sans-serif; color: #666666;">
+                                    If you have metastatic breast cancer and are interested in joining the research
```

Comment:
Can you do a find/replace on all new files for `breast cancer` -> `prostate cancer`?